cmd/gossamer/init.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/ChainSafe/gossamer/common"
+	"github.com/ChainSafe/gossamer/trie"
+)
+
+func loadTrie(t *trie.Trie, data []map[string]string) error {
```

Comment:
Perhaps these should be part of the trie package? Maybe use trie as the receiver

Reply:
perhaps, but the `data` is actually an array of maps (from `Raw` field in genesis file) it sort of suggests that there might be a way to insert data from genesis that isn't in a raw format later on? or maybe there will be multiple maps inserted differently? I could add the inner loop to the trie package though, that loads just one map. what do you think

Reply:
That because of how you've defined GenesisFields. It looks like you just need something like a `map[hash]bytes`, based on whats in the genesis right now.

Reply:
true, I just went off what parity's looked like. should I change it to just map or leave it as an array of maps?

Reply:
change it pls

Reply:
changed, also moved loadTrie to the trie package